javacord-api/src/main/java/org/javacord/api/interaction/InteractionBase.java

```diff
@@ -142,4 +144,10 @@ default CompletableFuture<Void> respondWithModal(String customId, String title,
      */
     Optional<DiscordLocale> getServerLocale();
 
+    /**
+     * Gets the set of permissions of the app in context to the interaction.
+     *
+     * @return The app's permissions, in context of the interaction.
+     */
+    Optional<EnumSet<PermissionType>> getAppPermissions();
```

Comment:
I think this should be renamed as we do not have anything else than a bot like an only REST-Client
```suggestion
    Optional<EnumSet<PermissionType>> getBotPermissions();
```